src/algebra/algebra/basic.lean

```diff
@@ -1359,13 +1408,15 @@ section int
 
 variables (R : Type*) [ring R]
 
+-- FIXME
```

Comment:
What should be fixed, or has it been fixed now?

Reply:
I think this is referring to the fact that we would need a `add_comm_group.rzsmul` field otherwise we end up with another diamond again.